homie-controller/src/lib.rs

```diff
@@ -172,9 +172,8 @@ impl HomieController {
     }
 
     async fn handle_event(&self, incoming: Incoming) -> Result<Option<Event>, PollError> {
-        log::trace!("Incoming: {:?}", incoming);
```

Comment:
is this trace not useful anymore?

Reply:
It's already logged just up above in `poll`.